src/random.jl

```diff
@@ -0,0 +1,120 @@
+function random_cl(;
+    start = Dates.DateTime(2020, 1, 1),
+    step = Dates.Minute(1),
+    length = 500,
+    price_init = nothing,
+    price_init_min = 1.00,
+    price_init_step = 0.01,
+    price_init_max = 1000.00,
+    price_var_min = -1.0,
+    price_var_step = 0.01,
+    price_var_max = 0.01,
```

Comment:
Is downward trend intentional? Maybe by default should be `price_var_max = 1.0`?

Reply:
Fixed